CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# Below is a list of Flutter team members who are suggested reviewers
+# for contributions to plugins in this repository.
+
+packages/cloud_firestore/* @collinjackson @kroikie
+packages/firebase_auth/* @collinjackson @kroikie
+packages/firebase_messaging/* @collinjackson @kroikie
+packages/camera/* @bparrishMines 
```

Comment:
@mklim ok to add you as an additional point of contact here?

Reply:
SGTM